compiler/rustc_hir/src/def.rs

```diff
@@ -75,7 +75,10 @@ pub enum DefKind {
     Const,
     /// Constant generic parameter: `struct Foo<const N: usize> { ... }`
     ConstParam,
-    Static(ast::Mutability),
+    Static {
+        /// Whether it's a `static mut` or just a `static`.
+        mt: ast::Mutability,
```

Comment:
`mutbl` would be better, because that's the common name for `Mutability` fields/variables. (`mutability` is also used, but I think `mutbl` is fine.) `mt` is used for `MutTy` or `TypeAndMut`.